React/tic-tac/src/app/components/Header/index.js

```diff
@@ -1,17 +1,47 @@
 import React, { Component } from 'react';
 import logo from '@assets/logo.svg';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import actions from '@redux/auth/actions';
 
 import style from './styles.scss';
 
 class Header extends Component {
   render() {
     return (
       <header className={style.appHeader}>
-        <img src={logo} className={style.appLogo} alt="logo" />
-        <h1 className={style.appTitle}>Tic Tac Toe</h1>
+        <div className={style.child} />
+        <div className={`${style.child} ${style.childfeatured}`}>
+          <img src={logo} className={style.appLogo} alt="logo" />
+          <h1 className={style.appTitle}>Tic Tac Toe</h1>
+        </div>
+        <div className={` ${style.child} ${style.alignright}`}>
+          <ul className={style.menu}>
+            <li>
+              <button type="submit" className={style.button} onClick={this.props.logOut}>
+                LogOut
+              </button>
+            </li>
+          </ul>
+        </div>
       </header>
     );
   }
 }
 
-export default Header;
+Header.propTypes = {
+  logOut: PropTypes.func
+};
+
+const MapStateToProps = state => ({
```

Comment:
`mapStateToProps`. Variables un pascalCase. Only components must be named in CamelCase